syntax = "proto3";

import "Task.proto";
import "TaskId.proto";

package task_manager;

message SolidTask
{
    TaskId task_id = 1;
    TaskId parent_id = 2;
    Task task = 3;
}